examples/lock-app/genio/src/LockManager.cpp.bak

```diff
@@ -0,0 +1,706 @@
+/*
```

Comment:
Did you really mean to check in this `.bak` file?  There are others in this PR too....

Reply:
Sorry, removed.

Reply:
There are still multiple .bak files in this PR...

Reply:
Sorry again, removed